scripts/update-latest-versions.js

```diff
@@ -32,6 +32,13 @@ const client = new S3Client({
   latestLinks.forEach(link => {
     link[0] = link[0].substring('nodejs/release/'.length);
     link[1] = link[1].substring('nodejs/release/'.length);
+
+    if (
```

Comment:
My question here is why this started to happen. I think you should debug the output of the listDirectory function, maybe that's where the thing is coming all weird.

Reply:
I'm pretty sure it's these lines, they've been there though so I'm not entirely sure why we're _just_ seeing this https://github.com/nodejs/nodejs-latest-linker/blob/371c586c7b245689a97ef6f6757404a80c318f75/common.js#L71-L72

Reply:
Wouldnt that be affecting all other lines?

Can you run the script locally and debug the output of listDirectory folder?